examples/gan/dcgan_tf_keras/export.py

```diff
@@ -25,9 +25,7 @@ def generate_and_plot_images(generator: tf.keras.Sequential, noise_dim: int) ->
 
 
 def export_model(experiment_id: int, master_url: str) -> tf.keras.Model:
```

Comment:
You will have to remove `master_url` as a parameter, and invoke `client.login()` somewhere (probably in main())